packages/blockchain-wallet-v4-frontend/src/components/Form/SelectBoxCoin/selectors.js

```diff
@@ -6,7 +6,7 @@ export const getCoins = (state, { type }) => {
   const supportedCoins = selectors.core.walletOptions
     .getSupportedCoins(state)
     .getOrFail()
-  const coinOrder = ['PAX', 'USDT', 'BTC', 'BCH', 'ETH', 'XLM']
+  const coinOrder = ['BTC', 'BCH', 'ETH', 'XLM', 'USDT', 'PAX']
```

Comment:
do we need ALGO here as well?

Reply:
I don't think we'll have algo here yet since its custodial only, but I do need to add to homepage